### PR TITLE
Fix CRISPRessoPooled headers error and update docs

### PR DESCRIPTION
**Pull request by Snicker7**
Bug fix for CRISPRessoPooled headers and documentation. Headers are now removed if any of the headers can be matched. The new header titles have been added to documentation in the README and the CRISPRessoPooled -h.

### CODE REVIEW

**Review by kclem — APPROVED** [review]
Looks good - we'll eventually want to pull the parameter descriptions from the json file..

File: CRISPResso2/CRISPRessoPooledCORE.py [hunk 1]
```diff
@@ -626,13 +637,15 @@ def main():
 
                 match = difflib.get_close_matches(long_head, amplicon_input_column_names, n=1)
                 if not match:
-                    has_header = False
+                    no_header_match = True
```
Comment (kclem) [inline CRISPResso2/CRISPRessoPooledCORE.py:640]: better variable name would be 'has_unmatching_header'

File: CRISPResso2/CRISPRessoPooledCORE.py [hunk 2]
```diff
@@ -616,7 +626,8 @@ def main():
             head_lookup['name'] = 'amplicon_name'
 
             headers = []
```
Comment (Colelyman) [inline CRISPResso2/CRISPRessoPooledCORE.py:628]: It seems like we may be able to delete this `headers = []` now, right?

Comment (Colelyman) [reply CRISPResso2/CRISPRessoPooledCORE.py:628]: Never mind!

**Review by Colelyman — APPROVED** [review]

**Review by kclem — APPROVED** [review]

**Pull request merged**